Assets/Mirror/Components/NetworkProximityChecker.cs

```diff
@@ -26,6 +26,9 @@ public enum CheckMethod
 
         [Tooltip("Enable to force this object to be hidden from players.")]
         public bool forceHidden;
+        
+        [Tooltip("Always keep local player on his own observer list.")]
+        public bool alwaysKeepLocalPlayerObserver;
```

Comment:
@rodolphito  you don't think this should be true by default?  

Is there ever any case where we want it to be false?   

Do we even need this option?

Reply:
I think if an option exists, the default value for it shouldn't change. Since this option didnt exist before, I'd say it should default to false, to be consistent with the original behaviour.

However, as you point out, the option may not be needed at all if the old behaviour is underiable. After all, we haven't added checkboxes to toggle bugfixes on and off before.

Reply:
A checkbox that essentially says "fix bug 692"  and is off by default makes no sense to me.